src/views/Profile/index.js

```diff
@@ -154,206 +191,21 @@ export default class Profile extends Component {
-                            {this.state.prefImage && (
-                              <img
-                                className="rounded-corners"
-                                src={`data:image/jpg;base64,${this.state.prefImage}`}
-                                alt=""
-                                style={{ maxHeight: '200px', minWidth: '160px' }}
-                              />
-                            )}
-                            {this.state.prefImage && this.state.defImage && ' '}
-                            {this.state.defImage && (
-                              <img
-                                className="rounded-corners"
-                                src={`data:image/jpg;base64,${this.state.defImage}`}
-                                alt=""
-                                style={{ maxHeight: '200px', minWidth: '160px' }}
-                              />
-                            )}
-                          </Grid>
-                        </Grid>
-                        <Grid item xs={12} sm={6} md={6} lg={4}>
-                          <Grid container align="center" alignItems="center" spacing={2}>
-                            <Grid item xs={12}>
-                              <CardHeader
-                                title={
-                                  this.state.hasNickName
-                                    ? this.state.profile.fullName +
-                                      ' (' +
-                                      this.state.profile.NickName +
-                                      ')'
-                                    : this.state.profile.fullName
-                                }
-                                subheader={this.state.subheaderInfo}
-                              />
-
-                              <Grid container spacing={2} align="center" justify="center">
-                                {facebookButton}
-                                {twitterButton}
-                                {linkedInButton}
-                                {instagramButton}
-                              </Grid>
-                              {this.state.profile.Email !== '' && (
-                                <a
-                                  href={`mailto:${this.state.profile.Email}`}
-                                  className="gc360-text-link gc360-profile_email"
-                                >
-                                  <EmailIcon className="gc360-profile_email_icon" />
-                                  <Typography className="gc360-profile_email_text">
-                                    {this.state.profile.Email}
-                                  </Typography>
-                                </a>
-                              )}
-                            </Grid>
-                          </Grid>
-                        </Grid>
-                      </Grid>
-                    </CardContent>
-                  </Card>
```

Comment:
All of the code from lines 157-356 is no longer needed as they've been replaced with the Identification component